### PR TITLE
Add tracing configs to frontend and recommendations service

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by dilanSachi** [comment]
@NipunaMadhushan maybe you can add your changes to the same pr

**Review by NipunaMadhushan — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+      - 55680:55680
   ads:
     image: wso2inc/ads-service:v0.1.0
   cart:
     image: wso2inc/cart-service:v0.1.0 #add redis
   checkout:
     image: wso2inc/checkout-service:v0.1.0
     environment:
-      BAL_CONFIG_DATA: catalogHost="catalog" \n cartHost="cart" \n currencyHost="currency" \n shippingHost="shipping" \n paymentHost="payment" \n emailHost="email"
+      BAL_CONFIG_DATA: catalogHost="catalog" \n cartHost="cart" \n currencyHost="currency" \n shippingHost="shipping" \n paymentHost="payment" \n emailHost="email" catalogTimeout=3.0 \n cartTimeout=3.0 \n currencyTimeout=3.0 \n shippingTimeout=3.0 \n paymentTimeout=3.0 \n emailTimeout=3.0
```
Comment (xlight05) [inline docker-compose.yml:17]: Can't we have the timeout 3.0 as the default value in configurables? Then we can reduce the lines here. If this BAL_CONFIG_DATA increases, the readability heavily gets reduce :( wdyt?

Comment (dilanSachi) [reply docker-compose.yml:17]: ack
will change

Comment (dilanSachi) [reply docker-compose.yml]: Added https://github.com/ballerina-guides/gcp-microservices-demo/pull/37/commits/e37e5cac02f8e9d418281a152d86b8c3a120eaf9

**Pull request merged**